app/controllers/users_controller.rb

```diff
@@ -5,12 +5,12 @@ def new
 
   def create
     @user = User.new(user_params)
-    folders = Folder.new
     if @user.save
```

Comment:
I changed how this view works so that we only have to pass through one variable to the controller.